packages/web/graphql/getQuests.ts

```diff
@@ -62,6 +62,17 @@ gql`
   }
 `;
 
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
@vidvidvid  This is not necessary in this PR but looks like it doesn't make a big difference.
But you may also wanna include changes in `packages/backend/src/handlers/actions/quests/createQuest/createQuest.ts` as well.

Reply:
also I'm not able to find changes related to `quest_completion` and `quest` table relationship that we discussed over chat. Would that be included in a future PR ?